.github/workflows/changelog.yml

```diff
@@ -1,26 +1,30 @@
-name: Changelog
+name: changelog
 
 on:
   pull_request:
-    types: [labeled, unlabeled, opened, synchronize, reopened]
+    types:
+      - labeled
+      - unlabeled
+      - opened
+      - synchronize
+      - reopened
 
-# Only cancel in-progress jobs or runs for the current workflow
-#   This cancels the already triggered workflows for a specific PR without canceling
-#   other instances of this workflow (other PRs, scheduled triggers, etc) when something
-#   within that PR re-triggers this CI
 concurrency:
   group: ${{ github.workflow }}-${{ github.ref }}
   cancel-in-progress: true
 
 jobs:
-  changelog:
-    name: Confirm changelog entry
+  check:
+    if: ${{ !contains(github.event.pull_request.labels.*.name, 'no-changelog-entry-needed') }}
```

Comment:
One last question related to these changes and the branch protection.

Our changelog check is listed as one of the branch protection rules (I am in the process of updating these so it might not show as required at the moment). I see this skip is at the `job` level now instead of the `step` and that the job shows as skipped. Previously when the `no-changelog-entry-needed` label was added the job still ran but the check was skipped (see this [run](https://github.com/asdf-format/asdf/actions/runs/10287686705/job/28471332037)) which I believe allowed the branch protection to pass. Will it still pass if the job is skipped (and not just the check)?

Reply:
yes indeed, as evidenced by this PR